### DISCUSSION TITLE
Getting on with development and update to actual openDTU code now and in sight of 2025 ...

### DISCUSSION THREAD

**Discussion by MartinNK** [RaBa64/OpenDTU#34] [Ideas]
Hi everybody,

I know how much time is to be spent for such a project ... keeping it uptodate and getting forward in development ... have ideas and a timeline for the future.

I don't know how many people ar ewatching here and using database version / Fork of @RaBa64 ... hope to see the userbase rising a little more.

Maybe we alle can manage to help Ralf at his work and develope ideas, code and testing a bit more!

@RaBa64 are you still on board and thinking to develop / update it in future? What about integrating your work in main openDTU version or the "on battery" fork?

Best regards for the future
Martin

**Comment by RaBa64** [comment]
Hello friends of OpenDTU-Database,
I am very sorry that I have so few time to keep this little project up to date. I am trying the best I can.
From time to time I update the project to the latest OpenDTU main branch.
Normally that is pretty easy and straight forward, but the changes in OpenDTU which were made about 2 month ago, need more attention and time. I hope I can do these changes during my Christmas holidays (although my wife has other plans for me).
Another problem is, that I still have not found a good replacement for Google Charts for the hourly view. Any suggestions are welcome.
Although I don't have a battery myself, I will have a look at the battery fork. It should be no problem to integrate the database, because the code is pretty separate.
I am not an expert for Vue, so all improvements here are always painfull for me, but I can learn a lot. If there is a Vue expert here, any help is appreciated.
Also If somebody could help me to fix the github build pipeline, please contact me (most probably also a Vue problem).

**Reply by MartinNK** [reply]
Hi Ralf, 

I hope you enjoy chrismas and had a good change to the year 2025, wishing you and your family all the best ... health, good luck and happiness for 2025.

Any news about opendtu database and new development.

Tanks and take care
Martin

**Reply by RaBa64** [reply]
Thank you! Happy new year to you and everynody else.
Unfortunately I had not so much time over Christmas to work on OpenDTU.
My plan is to make it run again with the latest OpenDTU release as soon as possible.

**Comment by MartinNK** [comment]
Better to have time with family and do work on openDTU and sit in front of computer ... later ;-) :-)If I could help just contact me. I will try my best.Best regards to everybody MartinVon meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: Ralf Bauer ***@***.***> Datum: 10.01.25  12:07  (GMT+01:00) An: RaBa64/OpenDTU-Database ***@***.***> Cc: MartinNK ***@***.***>, Author ***@***.***> Betreff: Re: [RaBa64/OpenDTU-Database] Getting on with development and update
  to actual openDTU code now and in sight of 2025 ... (Discussion #34) 
Thank you! Happy new year to you and everynody else.
Unfortunately I had not so much time over Christmas to work on OpenDTU.
My plan is to make it run again with the latest OpenDTU release as soon as possible.

**Comment by RaBa64** [comment]
I did some improvements on the build pipeline, so it will be much easier and faster to update to a new release of OpenDTU in the future. The merge with the latest main OpenDTU version is available now.

**Reply by MartinNK** [reply]
Thanks very much for your work. I will try it in the afternoon.

@RaBa64 ... haste Dir mal [Charts.js](https://www.chartjs.org/docs/latest/) angesehen?

**Reply by MartinNK** [reply]
@RaBa64 

Wie sieht eigentlich bei der neuen Version aus mit der Partitiontable?

War da nicht etwas mit der Reorganisation der Partitionen wegen der verschiedenen Flashgröße bei ESP32 mit 4, 8 oder 16 MB?
